tests/evaluate_sentence_splitter.py

```diff
@@ -0,0 +1,90 @@
+import re
+from collections.abc import Generator
+from pathlib import Path
+
+import spacy
+import typer
+from tqdm import tqdm
+
+from fun_sentence_splitter.sentence_splitter import Span, init
+
+WHITE_SPACE = re.compile(r"\s+")
+
+
+def main(
+        data_dir: Path,
+        max_len: int = 100,
+        spacy_model: str = "de_core_news_sm",
+) -> None:
+    sentence_splitter = init(
+        max_len_before_split=max_len,
+        spacy_model=spacy_model,
+    )
+
+    tp, fp, fn = 0, 0, 0
+    spans_count = 0
+    texts_count = 0
+    for split_file in tqdm(list(data_dir.glob("*.split"))):
+        text_file = split_file.with_suffix(".txt")
+        gold_spans = frozenset(_find_spans(text_file=text_file, split_file=split_file))
+        test_spans = frozenset(sentence.span for sentence in sentence_splitter(text_file.read_text()))
+        tp_, fp_, fn_ = _count_overlap(gold_spans, test_spans)
+        tp += tp_
+        fp += fp_
+        fn += fn_
+        spans_count += len(gold_spans)
+        texts_count += 1
+    weighted_avg_f1 = _f1(tp, fp, fn)
```

Comment:
I think we do not need f-score here. We must find all sentences so missing (false negatives) are not acceptable.
And as all wrong ones can mostly occur by misseng true ones, we might use
```suggestion
    weighted_avg_f1 = tp / spans_count *(1- fp / spans_count)
```
or even simpler (if we do not want to punish artifacts double)
```suggestion
    hit_rate = tp / spans_count
```